.github/workflows/main.yml

```diff
@@ -38,12 +38,22 @@ jobs:
         echo Using Bufbuild version $BUFVERSION
         curl -sSL https://github.com/bufbuild/buf/releases/download/$BUFVERSION/buf-$(uname -s)-$(uname -m) > buf
         chmod a+x buf
+
+    - name: Download Buildifier
+      run: |
+        BUILDIFIER="3.4.0"
+        echo Using Buildifier version $BUILDIFIER
+        curl -sSLO https://github.com/bazelbuild/buildtools/releases/download/$BUILDIFIER/buildifier > buildifier
+        chmod a+x buildifier
         
     - name: Kotlin lint check
       run: ./ktlint --android domain/src/**/*.kt utility/src/**/*.kt data/src/**/*.kt app/src/**/*.kt testing/src/**/*.kt && echo "Lint completed successfully"
     
     -  name: Protobuf lint check
        run: ./buf check lint --input=model/src/main/proto --input-config buf.yaml && echo "Protobuf lint check completed successfully"
+
+    - name: Bazel lint check
+      run: ./buildifier --lint=warn --mode=check --warnings=-native-android -r app data domain model testing utility tools BUILD.bazel WORKSPACE oppia_android_test.bzl
```

Comment:
as [er the official doc, we have the option to ignore any check using `-` and add any check using `+` but there is no option like `check everything except native android`, so currently `--warnings=-native-android` is not checking anything, just skipping the native android checks.